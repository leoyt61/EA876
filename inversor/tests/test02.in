MUITAS PALAVRAS NUMA GRANDE SEQUENCIA
