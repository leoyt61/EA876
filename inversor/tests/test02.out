SATIUM SARVALAP AMUN EDNARG AICNEUQES
